src/integrationtests/java/com/aws/greengrass/integrationtests/BridgeTest.java

```diff
@@ -28,6 +28,8 @@
 @BridgeIntegrationTest
 public class BridgeTest {
 
+    private static final long AWAIT_TIMEOUT_SECONDS = 30;
```

Comment:
bring this way down, should take like 2 sec?

Reply:
fixed